src/rendezvous/MSC3906Rendezvous.ts

```diff
@@ -25,11 +32,23 @@ import { logger } from "../logger";
 import { sleep } from "../utils";
 
 enum PayloadType {
```

Comment:
please could you give this a doc-comment with a bit text explaining what it does. (The name is *sort-of* self-explanatory, but I would prefer it to be explicit).

As I understand it it is used to populate the `type` property in the MSC3886 payload?